core/templates/dev/head/domain/exploration/StatesObjectFactorySpec.js

```diff
@@ -53,18 +54,34 @@ describe('States object factory', function() {
               content_id: 'default_outcome',
               html: ''
             },
-            param_changes: []
+            param_changes: [],
+            labelled_as_correct: false,
+            refresher_exploration_id: null,
+            missing_prerequisite_skill_id: null
           },
           hints: [],
+          solution: null,
           id: 'TextInput'
         },
         param_changes: [],
+<<<<<<< HEAD
```

Comment:
Ah, this is removed in next commits. So, it is still visible in this one.

Reply:
@seanlip I am not sure about this, because whenever such bad merge statements appear, they mess up with the syntax of the language and hence the functionality of the file. The author is required to test the changes locally which leads to the eventual removal of such statements. 
Although a precautionary check sounds no harm. Will file an issue for the same.
Thanks!

Reply:
@ankita240796 Was the linter turned off in this case too?

Reply:
Yes.